static/app/views/settings/components/settingsLayout.tsx

```diff
@@ -75,11 +75,11 @@ function SettingsLayout(props: Props) {
 
       <MaxWidthContainer>
         {shouldRenderNavigation && (
-          <SidebarWrapper isVisible={navVisible} offsetTop={navOffsetTop}>
-            {renderNavigation!()}
+          <SidebarWrapper isVisible={mobileVisible} offsetTop={navOffsetTop}>
+            {renderNavigation({mobileVisible})}
```

Comment:
Looks like we're passing new state in here but does anything use it?

Reply:
I'm using it in the test

Reply:
Oh, but i did this on the plane since I couldn't figure out if it was possible to change the resolution of the test browser to trigger the media query, let me google that now that I have internet haha.